### ISSUE TITLE
Add `no-duplicate-class` rule

### ISSUE THREAD

**Issue by yeonjuan** [yeonjuan/html-eslint#317]
It would be nice to add a rule to check if there are duplicate classes.
## Incorrect
```html
<button class="btn btn"> </button>
```

## correct

```html
<button class="btn"> </button>
```
